pyproject.toml

```diff
@@ -72,6 +72,7 @@ nominal = { workspace = true }
 
 [project.optional-dependencies]
 hdf5 = ["h5py>=3.0", "tables>=3.7"]
+protos = ["nominal-api-protos>=0.0.0"]
```

Comment:
oh, I just realized that doing this as an optional dependency means that if an existing user updates their nominal client, they likely won't specify `[protos]` and hence their ability to stream would break. 

we could keep around both implementations and raise a deprecated warning on the old one I guess?